feature/experimental/p4rt/internal/p4rtutils/p4rtutils.go

```diff
@@ -189,11 +193,38 @@ func explicitP4RTNodes() map[string]string {
 	}
 }
 
+// inferP4RTNodesCisco infers the P4RT node name from the port name and device model
+// for Cisco devices.
+func inferP4RTNodesCisco(t *testing.T, dut *ondatra.DUTDevice) map[string]string {
+	t.Helper()
+	npus := []int{0, 1, 2}
+	pranges := []int{11, 23, 35}
+	res := make(map[string]string)
+	dist := dut.Model() == "" || strings.HasPrefix(dut.Model(), "CISCO-88")
+	for _, p := range dut.Ports() {
+		if dist {
```

Comment:
I agree. The reliable way to do this is to look at component tree, but that adds 3-5 minutes to the test's running time, since the test can't access the rundata/dutInfo information that is collected prior to the test execution. I'm not sure if there are any plans to export that. For now, we will rely on the information in the testbed file.